cmds/wifi-ap/cmd_wifi-ap.go

```diff
@@ -72,14 +72,10 @@ func main() {
 		err = wifiAPClient.Disable()
 	case opts.enabled:
 		res, err := wifiAPClient.Enabled()
```

Comment:
here compiler interpret err is a new local var, i think. That's why ci is complaining. Declare res just the line before and use the = instead of :=